book/website/community-handbook/style/style-figures.md

```diff
@@ -1,34 +1,48 @@
-- If you are using your images, they will be by-default included in this book under CC-BY 4.0 License.
-- Cite the image properly as directed by the image owners, “Image from the internet” is not enough.
+In this sub-chapter we have used the term **figure** to refer to an illustration that conveys information in the context of _The Turing Way_ chapters.
+The term **image** is used to refer to the file object itself.
 
-*Please note that we have used the term 'figure' to refer to an illustration that conveys information in the context of _The Turing Way_ chapters and the term 'image' is used to refer to the file object.*
+## Open licence
 
-## Image type, file size and location:
+Please ensure that you attribute the image files fairly and avoid files that are either restricted from reuse or lack reproduction permissions.
 
-Please upload .jpg or .png files that are under 1MB to allow them to load faster in the online book.
+The following recommendations will help you to check that you're using the images according to their licence permissions:
+
+* If you are using your own images, please add them to the `figures` folder.
+  They will then be made available under a [Creative Commons Attribution 4.0 International (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/deed.ast) license as with the rest of the book.
+* Alternatively, you can source images in the public domain ([CC0 licence](https://creativecommons.org/share-your-work/public-domain/cc0)) or images licensed through appropriate Creative Commons terms.
+  Images that are also available under CC-BY 4.0 permissions are very easily interoperable with the _The Turing Way_ as this is the same license as the rest of the content for the book.
+* If an image (for example that you have found on the internet), is not available under an open licence please contact the original author of the image and seek permission to reproduce their image.
+  Make sure to ask them **how they would like to be credited** in the caption for the figure.
+
+In general, make sure to always cite the image properly as directed by the image owners.
+"Image from the internet" is not enough.
+
+## Image type, file size and location
+
+Please upload `.jpg` or `.png` files that are under 1MB to allow them to load faster in the online book.
 If your file is larger than 1MB, please use a local image editing tools, or online tool like [IMG2GO](https://www.img2go.com/compress-image) to compress your file.
 
 Every image file used in this book should be located in the file `_figure-list.md` in the directory `book/website/figures` of our [GitHub Repository](https://github.com/alan-turing-institute/the-turing-way/tree/master/book/website/figures).
 If you use a new image file, please add the file in the `figures` directory, and add details in the `_figure-list.md`.
 
-## Style for including figures in a chapter:
+## MyST syntax to add a figure to a _Turing Way_ chapter
```

Comment:
I gotcha. I found the way that this section was written was quite confusing because at a quick scan it looked like you could use the regular markdown syntax but I think we expressly do NOT want them to do that. So I guess I have a reasonably strong opinion that we want to call out that we require MyST syntax rather than "any syntax that works"... what do you think?

Reply:
Discussed in person - lets keep the MyST because @KirstieJane cares about this more than @malvikasharan 😅